src/tests/unit/tests/electron/views/screenshot/screenshot-container.test.tsx

```diff
@@ -0,0 +1,64 @@
+// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License.
+import { ScreenshotData } from 'common/types/store-data/unified-data-interface';
+import { BoundingRectangle } from 'electron/platform/android/scan-results';
+import { ScreenshotContainer, ScreenshotContainerProps } from 'electron/views/screenshot/screenshot-container';
+import { shallow } from 'enzyme';
+import * as React from 'react';
+
+describe('ScreenshotContainer', () => {
+    const basicScreenshotData: ScreenshotData = { base64PngData: 'test-base-64-png-data' };
+    const highlightBoxRectangles: BoundingRectangle[] = [
```

Comment:
nit: this has a pretty general name but is only used in a single test case; I think it might be a little easier to follow that one test case if this were moved there.

Reply:
Moved to the one test case